tools/topology/sof-apl-da7219.m4

```diff
@@ -34,47 +34,53 @@ include(`platform/intel/dmic.m4')
 # Schedule 48 frames per 1000us deadline on core 0 with priority 0
 PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
 	1, 0, 2, s32le,
-	48, 1000, 0, 0)
+	48, 1000, 0, 0,
```

Comment:
So if frames come from srate (from runtime params) and scheduling time (from topology) surely we don't need the frame size here ? (and it can be deleted from macros)

Reply:
removed in later commit and as suggested will merge it with another one.